src/dave_data/datapool/hotmaps/hotmaps_request.py

```diff
@@ -0,0 +1,337 @@
+import os
+import shutil
```

Comment:
It is better to import the function that you need than the whole package, e.g. "from shapely.geometry import shape". 

Please check also the imports below